roboplot/imgproc/image_analysis.py

```diff
@@ -432,3 +429,64 @@ def error(line_normal, line_origin, indices):
             error = 0.0
 
         return error
+
+        
+        
+def find_start_direction(img):
+    """
+    This function takes an image and determine shich direction the path leaves the centre from.
+    Note in cases where the start of th path is at an angle (see northnortheast and eastnortheast
+    test cases) the result is sub optimal however the method should still be able to cope with 
+    these systems by recognising that the path needs to be rotated. If this is not the case 
+    this method will need to be revisited.
+    
+    Args:
+        img: The image whose centre is at the centre of the gree triangle which has now been made
+        white.
+
+    Returns:
+        enum orientation.
+    """
+
+    # Extract centre of the image.
+
+    img_width = img.shape[0]
+    img_height = img.shape[1]
+
+    sub_array = img[int(img_width/3):int(2*img_width/3), int(img_height/3):int(2*img_height/3)]
```

Comment:
Have left but this is interesting - I have had warnings about this but perhaps that was for lists.